pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md

```diff
@@ -215,3 +215,23 @@ count = spark.sparkContext.parallelize(range(1, n + 1), partitions).map(f).reduc
 print("Pi is roughly %f" % (4.0 * count / n))
 spark.stop
 ```
+
+### Using BeakerX Magic
```

Comment:
I'd suggest renaming it to `Using BeakerX Kernel` and adding something like that as the first line below the heading:
```
BeakerX Kernel supports Spark magic which allows to configure and start Spark Session via UI.
```